react/week2/my-app/src/components/WatchCount.js

```diff
@@ -0,0 +1,21 @@
+import React, {useState, useEffect } from 'react'
+
+const WatchCount = () => {
+
+    const [ count, setCount ] = useState(0);
+    useEffect(() => {
```

Comment:
This should be setTimeout. Why?

Reply:
I think, because setInterval would call everysecond, But setTimeout would run only when the useEffect function is called.